app/helpers/samples_helper.rb

```diff
@@ -126,14 +126,22 @@ def parsed_samples_for_s3_path(s3_path, project_id, host_genome_id)
     s3_output, _stderr, status = Open3.capture3("aws", "s3", "ls", "#{s3_path}/")
     return unless status.exitstatus.zero?
     s3_output.chomp!
-    entries = s3_output.split("\n").reject { |line| line.include? "Undetermined" }.select { |line| line.include? "fast" }
+    entries = s3_output.split("\n").reject { |line| line.include? "Undetermined" }
     samples = {}
     entries.each do |file_name|
-      matched = /([^ ]*)_R(\d)_001.(fastq.gz|fq.gz|fastq|fq|fasta.gz|fa.gz|fasta|fa)\z/.match(file_name)
-      next unless matched
+      matched_paired = InputFile::BULK_FILE_PAIRED_REGEX.match(file_name)
+      matched_single = InputFile::BULK_FILE_SINGLE_REGEX.match(file_name)
+      if matched_paired
+        matched = matched_paired
+        read_idx = matched[2].to_i - 1
+      elsif matched_single
+        matched = matched_single
+        read_idx = 0
+      else
+        next
```

Comment:
what does this mean?   malformed input is skipped?

Reply:
files in the s3 folder that are not fastq/fasta files are skipped (e.g. metadata files)